lib/hal/xbox.c

```diff
@@ -68,7 +95,17 @@ void XLaunchXBEEx(const char *xbePath, const void *launchData)
 
     if (!xbePath) {
         launchDataPage->Header.dwLaunchDataType = LDT_LAUNCH_DASHBOARD;
-    } else {
+
+        // Redirect to xonlinedash.xbe as needed.
```

Comment:
nit: Start lowercase and avoid punctation at end of comment for consistency with surrounding code

Reply:
It's already inconsistent, the comment on line 85 is capitalized with punctuation. Removing the comment as it's obvious from context.